DemoBench error creating symbolic links on Windows 10

This exception stack trace was captured from the "developer relations" chat on Slack:
```Java
ava.security.PrivilegedActionException: java.nio.file.FileSystemException: C:\Users\Munetoshi Yamada\demobench\20170510083758\bankofjapan-explorer\plugins\bank-of-corda.jar: クライアントは要求された特権を保有していません。

at java.security.AccessController.doPrivileged(Native Method)
at com.sun.javafx.tk.quantum.GlassViewEventHandler.lambda$handleMouseEvent$354(GlassViewEventHandler.java:417)
at com.sun.javafx.tk.quantum.QuantumToolkit.runWithoutRenderLock(QuantumToolkit.java:389)
at com.sun.javafx.tk.quantum.GlassViewEventHandler.handleMouseEvent(GlassViewEventHandler.java:416)
at com.sun.glass.ui.View.handleMouseEvent(View.java:555)
at com.sun.glass.ui.View.notifyMouse(View.java:937)
at com.sun.glass.ui.win.WinApplication._runLoop(Native Method)
at com.sun.glass.ui.win.WinApplication.lambda$null$148(WinApplication.java:191)
at java.lang.Thread.run(Thread.java:748)

Caused by: java.nio.file.FileSystemException: C:\Users\Munetoshi Yamada\demobench\20170510083758\bankofjapan-explorer\plugins\bank-of-corda.jar: クライアントは要求された特権を保有していません。

at sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:86)
at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:97)
at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:102)
at sun.nio.fs.WindowsFileSystemProvider.createSymbolicLink(WindowsFileSystemProvider.java:585)
at java.nio.file.Files.createSymbolicLink(Files.java:1043)
at net.corda.demobench.explorer.Explorer$installApps$$inlined$list$lambda$1.accept(Explorer.kt:89)
at net.corda.demobench.explorer.Explorer$installApps$$inlined$list$lambda$1.accept(Explorer.kt:16)
at java.util.Iterator.forEachRemaining(Iterator.java:116)
at java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1801)
at java.util.stream.ReferencePipeline$Head.forEachOrdered(ReferencePipeline.java:590)
at net.corda.demobench.explorer.Explorer.installApps(Explorer.kt:85)
at net.corda.demobench.explorer.Explorer.open(Explorer.kt:35)
at net.corda.demobench.views.NodeTerminalView$configureExplorerButton$1.handle(NodeTerminalView.kt:120)
at net.corda.demobench.views.NodeTerminalView$configureExplorerButton$1.handle(NodeTerminalView.kt:37)
at com.sun.javafx.event.CompositeEventHandler.dispatchBubblingEvent(CompositeEventHandler.java:86)
at com.sun.javafx.event.EventHandlerManager.dispatchBubblingEvent(EventHandlerManager.java:238)
...
```
It appears that creating symbolic links is a privileged operation on Windows, and DemoBench should handle the case when the user is not configured with this privilege.

--- a/tools/demobench/src/main/kotlin/net/corda/demobench/explorer/Explorer.kt
+++ b/tools/demobench/src/main/kotlin/net/corda/demobench/explorer/Explorer.kt
@@ -92,6 +92,10 @@ class Explorer internal constructor(private val explorerController: ExplorerCont
                     Files.copy(path, destPath)
                 } catch(e: FileAlreadyExistsException) {
                     // OK, don't care ...
+                } catch (e: IOException) {
+                    // Windows 10 might not allow this user to create a symlink
+                    log.warn("Failed to create symlink '{}' for '{}': {}", destPath, path, e.message)
+                    Files.copy(path, destPath)
                 }
             }
         }

